docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -164,4 +147,25 @@ The OIDC provider returns an HTTP 40x error code.
 The client application is not properly configured in the API ML Gateway.
 
 **Solution**  
-Check that the `client_id` and `client_secret` configured in the API ML Gateway correspond to the `client_id` and `client_secret` of the client application as configured in the OIDC provider.
+Check that the URL `jwks_uri` contains the key for  OIDC token validation.
+
+
+:::tip
+API ML Gateway exposes a validate token operation which is suitable during the OIDC setup. The operation makes it possible to verify that OIDC token is trusted by API ML. Note that the accounts mapping step is not included in this operation.
+
+Use the following curl command to validate that the OIDC token is trusted by API ML:
```

Comment:
Please validate this statement about what the curl command does.

Reply:
If validation passed than HTTP 200 - `OK` otherwise  HTTP 40x error